src/MessageList/MessageList.vala

```diff
@@ -112,6 +114,14 @@ public class Mail.MessageList : Gtk.Box {
             tooltip_text = _("Mark Conversation")
         };
 
+        move_menu = new Menu ();
+
+        move_button = new Gtk.MenuButton () {
+            image = new Gtk.Image.from_icon_name ("folder", Gtk.IconSize.LARGE_TOOLBAR),
+            tooltip_text = _("Move Conversation to…"),
+            menu_model = move_menu
```

Comment:
```suggestion
            menu_model = move_menu,
            use_popover = false
```

Fixes padding issues with PopoverMenu in Gtk3

Reply:
Ah nice didn't know that!